src/com/google/javascript/jscomp/NodeTraversal.java

```diff
@@ -657,6 +659,41 @@ private void traverseFunction(Node n, Node parent) {
     popScope();
   }
 
+  /** Traverses a class. */
+  private void traverseClass(Node n, Node parent) {
+    Preconditions.checkState(n.getChildCount() == 3, n);
+    Preconditions.checkState(n.isClass());
+
+    final Node className = n.getFirstChild();
+    boolean isClassExpression = (parent != null)
```

Comment:
Done. Removed `(parent != null)`. This was copied from `traverseFunction()`, although I'm not sure when `FUNCTION` will have null parent either.